charts/stardog/files/log4j2.xml

```diff
@@ -17,37 +17,37 @@
 			<JsonLayout compact="true" eventEol="true" includeStacktrace="true" stacktraceAsString="true" />
 		</Console>
 
-		<File name="stardogAppender"
+		<!-- <File name="stardogAppender"
```

Comment:
wasn't sure if there was a reason you had these logging to a file, so I haven't outright removed them.

Reply:
Yup, I mean both the ${LOG_DIR}/${STARDOG_LOG}.log and "${LOG_DIR}/${ZOOKEEPER_LOG}.log"
a) i didn't see log rolling but maybe i missed it?
b) It's been a while since I used log4j2 but is this LOG_DIR set to stardog.home in a non-overrideable way?  <Property
    name=\"LOG_DIR\">${sys:stardog.home}</Property>\n\t\t 
if b then are those users you mentioning collecting logs from inside the container somehow? I'm not sure I see why you'd want a log file in k8s, unless maybe it's a high volume log that you want mounted to ssd? 

I suppose best approaches here are either we use log4j prop evaluation to turn it on and off, or use helm templating, or give a seam where a user can provide their own log4j2.xml?

Reply:
added a seam to allow user to supply their own